app/javascript/components/schedule-form/schedule-form-fields.js

```diff
@@ -0,0 +1,228 @@
+import { componentTypes } from '@@ddf';
+import {
+  actionChange, runChange, runOptions, subActionChange, loadZoneOptions, objectTypeChange,
+} from './helper';
+
+export const nameField = () => ({
+  component: componentTypes.TEXT_FIELD,
+  name: 'name',
+  label: __('Name'),
+  maxLength: 50,
+  isRequired: true,
+  autoFocus: true,
+});
+
+export const descriptionField = () => ({
+  component: componentTypes.TEXTAREA,
+  name: 'description',
+  label: __('Description'),
+  isRequired: true,
+  maxLength: 100,
+});
+
+export const activeField = () => ({
+  component: componentTypes.CHECKBOX,
+  name: 'enabled',
+  label: __('Active'),
+});
+
+export const actionField = (actionOptions, filterOptions, setData, data) => ({
+  component: componentTypes.SELECT,
+  id: 'action',
+  name: 'action_typ',
+  label: __('Action'),
+  initialValue: 'vm',
+  onChange: (value) => actionChange(value, filterOptions, setData, data),
+  isRequired: true,
+  options: actionOptions,
+});
+
+export const filterField = (setData, data) => ({
+  component: componentTypes.SELECT,
+  id: 'filter_type',
+  name: 'filter_type',
+  label: __('Filter'),
+  onChange: (value) => subActionChange(value, setData, data),
+  hideField: data.hideFilterType,
+  options: data.subActionOptions,
+});
+
+export const targetField = (data) => ({
+  component: componentTypes.SELECT,
+  id: 'target_type',
+  name: 'target_type',
+  hideField: data.hideTarget,
+  options: data.targetOptions,
+  placeholder: __('<Choose>'),
+  includeEmpty: true,
+});
+
+export const zoneField = (data) => ({
+  component: componentTypes.SELECT,
+  id: 'zone',
+  name: 'zone_id',
+  label: __('Zone'),
+  placeholder: __('<Choose>'),
+  includeEmpty: true,
+  hideField: data.hideAutomationFields,
+  loadOptions: loadZoneOptions,
+});
+
+export const plainField = (name, text, data) => ({
+  component: 'plain-text',
+  name,
+  label: __(text),
```

Comment:
Is plainField only used https://github.com/ManageIQ/manageiq-ui-classic/pull/8606/files#diff-ce506f97c35dc4810fc3b83c0951665ca92d0a85f4e5af22760becd19fbbedf2R23 here? Because if so you should do:
`plainField('object_details', __('Object Details'), data), ` and `label: text,` instead

I
Do we have the list of possible strings `text` will be for the translation team?